src/Sarif.UnitTests/ConvertToSchemaUriTests.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
This is a VS option. Go to Tools, Options. In the Options dialog, in the tree view on the left, select the node Text Editor, C#, Advanced. In the right-hand pane, in the "Organize Usings" section, check the box for "Place 'System' directives first when sorting usings".